.github/workflows/ios-ci.yml

```diff
@@ -360,7 +366,7 @@ jobs:
             -d '{"ref":"main","inputs":{
               "changelog_url": "https://maplibre-native.s3.eu-central-1.amazonaws.com/changelogs/ios-${{ env.version }}.md",
               "version":"${{ env.version }}",
-              "download_url": "https://github.com/maplibre/maplibre-native/releases/download/ios-v${{ env.version }}/MapLibre.dynamic.xcframework.zip"'
+              "download_url": "https://github.com/maplibre/maplibre-native/releases/download/ios-v${{ env.version }}/MapLibre.dynamic.xcframework.zip"}}'
```

Comment:
Extra `}}`, intentional?

Reply:
Yes... That is an unrelated but needed fix.